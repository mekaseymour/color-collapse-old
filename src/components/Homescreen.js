import React from 'react';
import { StyleSheet, Text, View, Button } from 'react-native';

const Homescreen = () => (
  <View style={styles.container}>
    <Text>This is the Home Screen</Text>
    <Button onPress={() => {}} title="PLAY" color="#1194f6" />
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default Homescreen;
