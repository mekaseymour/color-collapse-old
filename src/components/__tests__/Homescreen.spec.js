import React from 'react';
import Homescreen from '../Homescreen';
import { configure, shallow } from 'enzyme';
import ReactSixteenAdapter from 'enzyme-adapter-react-16';

configure({ adapter: new ReactSixteenAdapter() });

it('renders without crashing', () => {
  const wrapper = shallow(<Homescreen navigation={{}} onPress={() => {}} />);
  expect(wrapper).toMatchSnapshot();
});
