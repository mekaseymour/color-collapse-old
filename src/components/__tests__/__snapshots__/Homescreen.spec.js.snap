// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 30,
      }
    }
  >
    Color Collapse
  </Text>
  <Button
    color="#1194f6"
    onPress={[Function]}
    title="PLAY"
  />
</Component>
`;
